modules/ROOT/pages/genai-integrations.adoc

```diff
@@ -266,19 +266,56 @@ Supported values:
 
 | model
 | STRING
-| The ID of the model you want to invoke. +
-Supported values: `"text-embedding-ada-002"`
-| "text-embedding-ada-002"
+| The name of the model you want to invoke.
+| `"text-embedding-ada-002"`
+
+| dimensions
+| INTEGER
+| The number of dimensions you want to reduce the vector to. Only supported for certain models (see official documentation).
```

Comment:
I would either be more specific of which models this applies to, or delete the bracketed content - which isnt telling the user a lot. 

```suggestion
| The number of dimensions you want to reduce the vector to. Only supported for certain models.
```

Reply:
I'd like to avoid being too specific as we don't have any control of any changes being made on the OpenAI side, and I think this page should avoid documenting OpenAI rather than the plugin (but that line is blurry at times).

Would a direct link to the relevant part of the OpenAI docs be better?

Reply:
Either that, or just simply "Only supported for certain models" -

Reply:
All righty 👍